nixos/modules/services/web-servers/caddy.nix

```diff
@@ -33,7 +40,7 @@ in
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
       serviceConfig = {
-        ExecStart = "${pkgs.caddy}/bin/caddy -conf=${configFile} -email=${cfg.email}";
+        ExecStart = "${pkgs.caddy}/bin/caddy -conf=${configFile} -ca=${cfg.ca} -email=${cfg.email} -agree";
```

Comment:
I think we should rather expose agreeing to the Let's Encrypt Subscriber Agreement as an option that defaults to false to be safe on the legal side of things.